lib/scholar/cluster/optics.ex

```diff
@@ -0,0 +1,207 @@
+defmodule Scholar.Cluster.OPTICS do
+  @moduledoc """
+  OPTICS (Ordering Points To Identify the Clustering Structure), closely related to DBSCAN, finds core sample of high density and expands clusters from them. Unlike DBSCAN, keeps cluster hierarchy for a variable neighborhood radius. Clusters are then extracted using a DBSCAN-like method.
```

Comment:
The first sentence should be really short.

Also please add a new line so it wraps around 80-100 characters. :)